devtools-ci-index.yaml

```diff
@@ -487,6 +487,14 @@
                 credential-id: 2fc94f6b-dfef-49a4-a217-fe47aaeea056
                 variable: RECOMMENDER_API_TOKEN
 
+- wrapper:
+    name: recommender_api_token_prod
+    wrappers:
+        - credentials-binding:
+            - text:
+                credential-id: 60ae2fd0-5ab0-4a12-bf75-6a81ab1b915f
+                variable: RECOMMENDER_API_TOKEN
```

Comment:
Is this already added to Jenkins on ci.centos.org?

Reply:
@aditya-konarde it's used here: https://github.com/almighty/almighty-jobs/blob/master/devtools-ci-index.yaml#L1979

I am assuming that it's there.

Reply:
Any reason why you are not referring to this as `credential-id: "{oso_token_creds}"` so we don't spread this key around in different ways ?

And why is exact same env variable defined above with  a different key ? (2fc9.. instead of 60ae..) ?

Reply:
IMHO the first is for stage instance, and the second for production one

Reply:
okey - so if this code actually used variables we could remove ~10 lines or so ;)

anyways, thats for a seperate PR. going to merge.

Reply:
@maxandersen the file indeed needs some general cleanup and refactoring. :)